packages/fluentui/accessibility/src/behaviors/Dropdown/dropdownSelectedItemBehavior.ts

```diff
@@ -10,11 +10,11 @@ import { Accessibility } from '../../types';
 export const dropdownSelectedItemBehavior: Accessibility<DropdownSelectedItemBehaviorProps> = props => ({
   attributes: {
     root: {
-      'aria-selected': true,
+      role: 'option',
       tabIndex: props.active ? 0 : -1,
     },
     icon: {
-      'aria-label': `Remove ${props.header} from selection.`,
+      'aria-hidden': 'true',
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/92102287-b1384900-edde-11ea-8034-ba10b5c3780a.png)

Are we hiding this element from narration?

Reply:
Yes hiding from narration by purpose, I don't want to narrate this icon at all, because user need to press Delete or Backspace key to remove selected item. And this instruction message I set to the element with role="listbox"